content/open-source/sigstore/cosign/how-to-sign-an-sbom-with-cosign.md

```diff
@@ -128,27 +128,6 @@ ExternalRef: PACKAGE_MANAGER purl pkg:alpine/zlib@1.2.12-r0?arch=aarch64&upstrea
 
 Next, you’ll use Cosign to work with the SBOM and the image.
```

Comment:
Is it worth linking out to cosign docs directly, as they're likely to be more up to date and then don't have to keep trying to keep this in synch? https://docs.sigstore.dev/cosign/signing/other_types/

Reply:
I'll raise this with the team. :+1: There are some upsides to maintaining our own docs on tools we use frequently or that are also used a lot in the secure container space (good for our SEO and pipeline). But the downside is that things can get out of sync. We're actually working on a new docs update cadence that should roll out soon as well.  cc @sheesh

Reply:
For sure yeah, two sides of the coin 😄 @smythp 

Will keep my eyes open for new docs cadence